src/modules/cli/lib/install-windows.ts

```diff
@@ -0,0 +1,107 @@
+import path from 'path';
+import Registry from 'winreg'; // don't update winreg to 1.2.5 - https://github.com/fresc81/node-winreg/issues/65
+import * as Sentry from '@sentry/electron/main';
+import { __ } from '@wordpress/i18n';
+import { mkdir, writeFile } from 'fs/promises'
+
+//Example of process.execPath - C:\Users\<USERNAME>\AppData\Local\studio\app-1.3.9-beta1\Studio.exe
+const localAppBinPath = path.resolve(process.execPath, '../../bin');
```

Comment:
```suggestion
// `unversionedBinDirPath` resolves to C:\Users\<USERNAME>\AppData\Local\studio\bin
const unversionedBinDirPath = path.resolve( path.dirname( app.getPath( 'exe' ) ), '../bin' );
```
A few things:
1. I think a variable name like this is a lot clearer.
2. `app.getPath` is the official Electron API for retrieving directories related to the current installation.
3. Using `path.dirname` isn't required, it just clarifies that we we expect the final segment of the path to point at a file rather than a directory. 
4. Not super important, but I think it's preferable to write the resulting path in the comment as opposed to the path returned from `app.getPath`

Reply:
>I think a variable name like this is a lot clearer.

What about `binDirPath`? I think that mentioning `unversioned` is a bit extra and just confusing.

>Using path.dirname isn't required, it just clarifies that we we expect the final segment of the path to point at a file rather than a directory.

Yeah, initially I considered adding `path.dirname` but decided that avoid using it make code more readable. But I don't have strong preference, so let's keep it 👌

Reply:
> What about binDirPath?

There are multiple `bin` dirs, which is why I think it's helpful to distinguish between them with the "versioned" and "unversioned" labels.

Reply:
Oh, yeah, you are right 👍
Updated